README

```diff
@@ -50,6 +50,7 @@ options:  --help                show this
                                 is process_vm_readv
           --ptrace              use libunwind-ptrace interface (slower)
           --show-rsp            show %rsp in second column
+          --show-state          show thread state before freeze
```

Comment:
Just `"show thread states"`?

Reply:
I wanted to draw user's attention on uncertainty of the reported state. It's very possible that the thread changes its state by the moment all threads are frozen and stack contents are read. It may be confusing to see, say, `select()` on top when a thread is marked as running (and this can be valid though).

Reply:
Well, I think it's obvious that getting state of the running thread is "racy".

Reply:
Ok, I don't insist. Added two fixups: https://github.com/tbricks/tbstack/pull/14/commits/b579af85d6bc3e5523c1db8dda829ca54ec5291b (help in .c) and https://github.com/tbricks/tbstack/pull/14/commits/0de055b0b40dabef98cd23c9d017a6900be33646 (help in README).